### PR TITLE
[REVIEW] Register `percentile_lookup` for `FrameProxyObject`

### PR DESCRIPTION
**Pull request by galipremsagar**
Fixes: #714 

This PR registers `percentile_lookup` for `FrameProxyObject`

### CODE REVIEW

**Comment by quasiben** [comment]
Thanks for doing this @galipremsagar !  This looks good to me but it would be good to get @madsbk thoughts when he is back on Monday

**Comment by quasiben** [comment]
rerun tests

File: dask_cuda/proxify_device_objects.py [hunk 1]
```diff
@@ -264,3 +264,12 @@ def proxify_device_object_cudf_dataframe(
         return proxify(
             obj, proxied_id_to_proxy, found_proxies, subclass=FrameProxyObject
         )
+
+    try:
+        from dask.array.dispatch import percentile_lookup
+
```
Comment (madsbk) [inline dask_cuda/proxify_device_objects.py:270]:
```suggestion
```

**Review by madsbk — APPROVED** [review]
LGTM, thanks @galipremsagar!

File: dask_cuda/proxify_device_objects.py (hunk 1 above)
Comment (galipremsagar) [reply dask_cuda/proxify_device_objects.py:270]: Looks like `isort` isn't happy here without a space, hence had to revert this change

**Comment by galipremsagar** [comment]
@gpucibot merge

**Comment by galipremsagar** [comment]
@gpucibot merge

**Comment by pentschev** [comment]
@gpucibot merge

**Comment by pentschev** [comment]
Thanks @galipremsagar !

**Pull request merged**